app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -634,6 +636,11 @@ public void pauseCompositor() {
             return;
         }
 
+        // When using layers, ensure that the surface is not left in an inconsistent state.
+        if (isLayer() && mSurface != null) {
```

Comment:
This change should only be applied when we are using layers, otherwise the native UI is not drawn.